### PR TITLE
[DEV-11014] - updated client to support new access request status

### PR DESCRIPTION
**Pull request by tomerstein**

### CODE REVIEW

File: pkg/services/requests.go [hunk 1]
```diff
@@ -18,7 +18,15 @@ import (
...
-	requestWaitingForApprovalStatus = "Pending Approval"
+	AccessRequestInitStatus               = "Initializing"
+	AccessRequestPendingStatus            = "Pending"
+	AccessRequestGrantingStatus           = "Granting"
+	AccessRequestRejectedStatus           = "Rejected"
+	AccessRequestActiveStatus             = "Active"
+	AccessRequestRevokingStatus           = "Revoking"
+	AccessRequestRevokedStatus            = "Revoked"
+	AccessRequestFailedStatus             = "Failed"
+	AccessRequestWaitingForApprovalStatus = "Pending Approval"
```
Comment (DmitriyO) [inline pkg/services/requests.go:29]: Pending and Pending Approval are the same one
Actual value returned is "Pending"
if you want to show user "Pending approval" you need to ma it everywhere
But not sure I would go this way

File: pkg/services/requests.go [hunk 2]
```diff
@@ -179,22 +187,22 @@ func RevokeRequest(ctx context.Context, client *aponoapi.AponoClient, requestID
...
 	}
 
-	statusTitle := cases.Title(language.English).String(string(status))
+	statusTitle := cases.Title(language.English).String(status)
 	switch status {
-	case requestWaitingForApprovalStatus:
+	case AccessRequestWaitingForApprovalStatus:
 		return color.HiYellow.Sprint(statusTitle)
-	case clientapi.ACCESSSTATUS_PENDING:
+	case AccessRequestInitStatus, AccessRequestPendingStatus:
```
Comment (DmitriyO) [inline pkg/services/requests.go:197]: AccessRequestWaitingForApprovalStatus and AccessRequestPendingStatus are same one

**Review by DmitriyO — APPROVED** [review]
Note the small changes to do

File: pkg/services/requests.go (hunk 2 above)
Comment (tomerstein) [reply pkg/services/requests.go:197]: thats why they have the same color

File: pkg/services/requests.go (hunk 1 above)
Comment (tomerstein) [reply pkg/services/requests.go:29]: I do it with the `ColoredStatus` function

**Pull request merged**